test/runtime/memcache.go

```diff
@@ -32,11 +32,13 @@ var _ = Describe("RuntimeMemcacheBinary", func() {
 	containers := func(mode string) {
 
 		images := map[string]string{
-			"memcache":        "memcached",
-			"memcache-client": "nebril/python-binary-memcached",
+			"memcache":             "memcached",
+			"memcache-bin-client":  "nebril/python-binary-memcached",
```

Comment:
Not blocking - create a GH issue to have this container built under the `cilium` organization. This should also be pulled into the VMs in the packer-ci-build repo. Can you do that as well?

Reply:
+1

Reply:
https://github.com/cilium/cilium/issues/5769